examples/cloudevents-abuse-protection/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+use anyhow::Result;
+use spin_sdk::{
+    http::{Request, Response},
+    http_component,
+};
+
+/// A spin component that can be used to test the event-grid-validation.
+/// For more info, see here: https://github.com/cloudevents/spec/blob/v1.0/http-webhook.md#42-validation-response
+#[http_component]
+fn validation(req: Request) -> Result<Response> {
```

Comment:
This doesn't seem to check for the `OPTIONS` verb - is that intentional?  Is the options support because options messages need to get through, but we don't actually care what the verb is?

Reply:
Good question. Actually this example will need to use the OPTIONS verb to distinguish a validation request vs. normal request. I will push an update.